explainer.md

```diff
@@ -269,5 +271,6 @@ The following mitigations are recommended.
 ## References:
 * [Ballista (earlier, related proposal) explainer](https://github.com/chromium/ballista/blob/master/docs/explainer.md).
 * Chrome Design Documents:
+  * [File Handling design document](https://tinyurl.com/file-handling-design)
```

Comment:
Do we have an established pattern of using tinyurl?  If so, great.  Otherwise if this is the first then I'd like to consider how to avoid stale links.  Other links below point directly to google docs so that may be preferable.

Reply:
pointed directly to google docs now